client/me/help/help-contact-closure-notice/index.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * Fixed notice about upcoming support closures
+ */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n, { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import olarkStore from 'lib/olark-store';
+
+const closedFrom = i18n.moment( 'Thu, 24 Nov 2016 08:00:00 +0000' );
+const closedTo = i18n.moment( 'Fri, 25 Nov 2016 08:00:00 +0000' );
+
+const Closed = localize( ( { translate } ) =>
+	<div className="help-contact-closure-notice">
+		<FormSectionHeading>{ translate( 'Limited Support For Thanksgiving' ) }</FormSectionHeading>
+		<p>
+			{ translate(
+				'Live chat is closed today for the US Thanksgiving holiday. To get in touch, please ' +
+				'submit a support request below and we will get to it as fast as we can. Live chat ' +
+				'will reopen on {{strong}}%(closed_end_date)s{{/strong}}. Thank you!', {
+					args: {
+						closed_end_date: closedTo.format( 'dddd, MMMM Do, YYYY HH:mm' ),
+					},
+					components: {
+						strong: <strong />
+					}
+				}
+			) }
+		</p>
+	</div>
+);
+
+const Upcoming = localize( ( { translate } ) =>
+	<div className="help-contact-closure-notice">
+		<FormSectionHeading>{ translate( 'Limited Support For Thanksgiving' ) }</FormSectionHeading>
+		<p>
+			{ translate(
+				'Live chat support will be closed from ' +
+				'{{strong}}%(closed_start_date)s{{/strong}} through {{strong}}%(closed_end_date)s{{/strong}} ' +
+				'for the US Thanksgiving holiday. If you need to get in touch with us that day, you’ll be able ' +
+				'to submit a support request from this page and we will get to it as fast as we can. Thank you!', {
+					args: {
+						closed_start_date: closedFrom.format( 'dddd, MMMM Do, YYYY HH:mm' ),
+						closed_end_date: closedTo.format( 'dddd, MMMM Do, YYYY HH:mm' ),
+					},
+					components: {
+						strong: <strong />
+					}
+				}
+			) }
+		</p>
+	</div>
+);
+
+export default class HelpContactClosureNotice extends Component {
+	constructor() {
+		super();
+		this.state = { olark: olarkStore.get() };
```

Comment:
I had thought about that, but as you mentioned that would introduce data into an extra intermediary component (HelpContactForm) which doesn't otherwise need that data. Ideally we'd have this data in the redux store eventually anyway, in which case I'd just use a `connect` on HelpContactClosureNotice.